tests/test_00_unit/test_00_index_reader/__init__.py

```diff
@@ -0,0 +1,128 @@
+import unittest
+
+from ccp import index_reader
+
+
+class TestProject(unittest.TestCase):
+    """
+    Test the Project class representing an
+    object in container pipeline
+    """
+
+    def setUp(self):
+        self.entry = {
+            "id": "1",
+            "app-id": "foo",
+            "job-id": "bar",
+            "desired-tag": "latest",
+            "git-branch": "master",
+            "git-url": "https://github.com/someurl",
+            "git-path": "foo/",
+            "target-file": "Dockerfile",
+            "build-context": "./",
+            "notify-email": "abc@example/com",
+            "depends-on": ["centos/centos:latest", "centos/centos:7"],
+            "prebuild-script": "hooks/script.sh",
+            "prebuild-context": "/",
+        }
+        self.namespace = "ccp"
+
+    def test_load_entries(self):
```

Comment:
Makes sense, this is more of a setup function

Reply:
Asserting is implicit in this case, meaning, if raise exception in loading the entries into object, the test fails. However, I agree that this could be done via some assertion, for eg: ensure it doesn't raise any exception.

@mohammedzee1000 : No, I thought about having  it setup funciton, however, it would run before every test, which I wanted to test explicitly as I am changing the entry values and loading the entry again.